contrib/test/ci/setup.yml

```diff
@@ -5,7 +5,7 @@
 - name: install Golang tools
   include: "golang.yml"
   vars:
-    version: "1.19"
+    golang_version: "1.19"
```

Comment:
I don't think we need this since we use `{{ golang_version }}` directly in golang.yml.

Reply:
Yup, good catch!